_source/logzio_collections/_prometheus-sources/python-custom-prometheus.md

```diff
@@ -215,4 +233,186 @@ meter.register_valueobserver(
 ##### Check Logz.io for your metrics
 Give your data some time to get from your system to ours, then log in to your Logz.io Metrics account, and open [the Logz.io Metrics tab](https://app.logz.io/#/dashboard/metrics/).
 
-</div>
\ No newline at end of file
+</div>
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="lambda">
+
+This integration uses OpenTelemetry collector extention and Python metrics SDK to create and send metrics from your Lambda functions to your Logz.io account.
+
+<!-- info-box-start:info -->
+This integration is currently only supported in the following AWS regions: **us-east-1**, **us-east-2**,**us-west-1**, **us-west-2**, **ca-central-1**, **ap-northeast-2**, **ap-northeast-1**,**eu-central-1**, **eu-west-2**. Contact Logz.io Customer Support if you need to deploy in a different region.
+{:.info-box.note}
+<!-- info-box-end -->
+
+#### Quick start
+
+
+<div class="tasklist">
+
+##### Create Lambda function
+
+Create a new Lambda function in your AWS account (with Python version >= 3.7) or use our example [deployment package](https://logzio-aws-integrations-us-east-1.s3.amazonaws.com/aws-otel-lambda-python/logzio-lambda-python-metrics-deployment.zip) including the code sample.
```

Comment:
Updated link to deployment .zip file -> https://logzio-aws-integrations-us-east-1.s3.amazonaws.com/aws-otel-lambda-python/logzio-python-lambda-custom-metrics-deployment.zip